ivo-projects/todo/src/components/Reminders.js

```diff
@@ -1,35 +1,57 @@
 import React from 'react';
 import moment from 'moment-timezone';
+import { FaPencil, FaCheck } from 'react-icons/lib/fa'
 
 class RemindersList extends React.Component {
-    componentWillMount () {
-        clearInterval(this.interval);
-    }
-    componentDidMount () {
-        this.interval = setInterval(
-            this.forceUpdate.bind(this),
-            parseInt(this.props.updateInterval, 10) || 100
-        );
+
+
+    renderEditButtons(reminder) {
```

Comment:
Create an `EditButtons` component and use it like:

<EditButtons reminder={reminder} />

EditButtons should be a stateless component and you will have in props the reminder.

Do the same for InputField: <InputField reminder={reminder} />